lib/auth/auth.go

```diff
@@ -782,6 +782,7 @@ func (a *AuthServer) createOIDCUser(connector services.OIDCConnector, ident *oid
 	if len(roles) == 0 {
 		log.Warningf("[OIDC] could not find any of expected claims: %v in the set returned by provider %v: %v",
 			strings.Join(connector.GetClaims(), ","), connector.GetName(), strings.Join(services.GetClaimNames(claims), ","))
+		return trace.AccessDenied("access to %v is denied", ident.Email)
```

Comment:
the error message does not seem right. Looks to me it will say `access to ev@kontsevoy.com is denied`. Is that correct? In that case it should change to `access denied to ev@kontsevoy.com`